src/shaders/MaterialX/build_materialX_osl.py

```diff
@@ -195,29 +195,38 @@
 
 BUILD_DICT = {
     'mx_absval': ALL_TYPES,
+    'mx_acos' : ALL_TYPES,
     'mx_add': ALL_TYPES + ['surfaceshader'],
     'mx_add_float': ['color', 'color2', 'color4', 'vector', 'vector2', 'vector4'],
+    'mx_asin' : ALL_TYPES,
+    'mx_atan2' : ALL_TYPES,
+    'mx_atan2_float' : ALL_TYPES,
     'mx_ambientocclusion': ['float'],
     'mx_bitangent': ['vector'],
     'mx_blur': ALL_TYPES,
     'mx_burn': ['float', 'color', 'color2', 'color4'],
+    'mx_ceil' : ALL_TYPES,
     'mx_cellnoise2d': ['float'],
     'mx_cellnoise3d': ['float'],
     'mx_clamp': ALL_TYPES,
     'mx_clamp_float': ['color', 'color2', 'color4', 'vector', 'vector2', 'vector4'],
     'mx_compare': ALL_TYPES,
     'mx_constant': ALL_TYPES + ['matrix44', 'matrix33', 'string', 'filename', 'bool', 'int'],
+    'mx_cos': ALL_TYPES,
     'mx_crossproduct': ['vector'],
+    'mx_determinant' : ['matrix44', 'matrix33'],
     'mx_dodge': ['float', 'color', 'color2', 'color4'],
     'mx_dotproduct': ['vector', 'vector2', 'vector4'],
     'mx_dot': ALL_TYPES + ['matrix44', 'matrix33', 'string', 'filename', 'bool', 'int', 'surfaceshader'],
+    'mx_exp' : ALL_TYPES + ['int'],
     'mx_floor': ALL_TYPES,
     'mx_frame': ['float'],
     'mx_geomattrvalue': ALL_TYPES + ['bool', 'string', 'int'],
     'mx_geomcolor': ['float', 'color', 'color2', 'color4'],
     'mx_heighttonormal': ['vector'],
     'mx_hueshift': ['color', 'color4'],
     'mx_image': ALL_TYPES,
+    'mx_ln' : ALL_TYPES + ['int'],
```

Comment:
This may be more of a question for Doug, but: "ln" vs "log"? What math calls ln, most programming languages call log. Are we intentionally being different from C/etc here?